### PR TITLE
New: (Lidarr/Radarr/Readarr/Sonarr) Improved Errors

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Warn users that a base url may be missing for \*Arrs
- Improved Validation Errors for \*Arr Tests
- Add Sonarr v2 note

#### Todos
- [ ] Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Closes https://github.com/Prowlarr/Prowlarr/issues/502

### CODE REVIEW

File: src/NzbDrone.Core/Applications/Radarr/RadarrV3Proxy.cs [hunk 1]
```diff
@@ -116,6 +116,12 @@ public ValidationFailure TestConnection(RadarrIndexer indexer, RadarrSettings se
                     return new ValidationFailure("ProwlarrUrl", "Prowlarr url is invalid, Radarr cannot connect to Prowlarr");
                 }
 
+                if (ex.Response.StatusCode == HttpStatusCode.SeeOther)
```
Comment (Qstick) [inline src/NzbDrone.Core/Applications/Radarr/RadarrV3Proxy.cs:119]: Don't we handle these now, at least I thought we did in some arrs?

<https://github.com/Radarr/Radarr/commit/88bc44f7ba7cd51c9e03065b472455e86eaa6d8e>

Comment (bakerboy448) [reply src/NzbDrone.Core/Applications/Radarr/RadarrV3Proxy.cs:119]: Maybe it's just sonarr that we don't like following a redir?

Comment (Qstick) [reply src/NzbDrone.Core/Applications/Radarr/RadarrV3Proxy.cs:119]: Idk, I'd have to test

**Comment by Qstick** [comment]
Can you add the 5th, then good.

**Review by Qstick — APPROVED** [review]

**Pull request merged**